common/main/he.xml

```diff
@@ -8636,13 +8636,13 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 			<pattern type="range">{0}–{1}</pattern>
 		</miscPatterns>
 		<minimalPairs>
-			<pluralMinimalPairs count="one">שנה</pluralMinimalPairs>
-			<pluralMinimalPairs count="two">שנתיים</pluralMinimalPairs>
+			<pluralMinimalPairs count="one">השנה האחרונה הייתה קשה</pluralMinimalPairs>
```

Comment:
Isn't 'שנה' the word for 'year'?

Reply:
Okay, I just thought the unit wasn't supposed to be in the phrase, but instead was intended to be represented by the placeholder?